app/src/androidTest/java/com/duckduckgo/app/browser/animations/LottiePrivacyShieldAnimationHelperTest.kt

```diff
@@ -14,19 +14,18 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.privacy.dashboard.impl.animations
+package com.duckduckgo.app.browser.omnibar.animations
```

Comment:
The package or the location of the file is wrong.

`package com.duckduckgo.app.browser.omnibar.animations` but the file is in `com.duckduckgo.app.browser.animations`